docs/sources/panels-visualizations/visualizations/pie-chart/index.md

```diff
@@ -35,6 +35,21 @@ The following video guides you through the creation steps and common customizati
 
 {{< docs/play title="Grafana Bar Charts and Pie Charts" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
 
+## Supported data formats
+
+The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.
+
+Example:
+
+| Description   | Internet Explorer | Chrome |
+| ------------- | ----------------- | ------ |
+| Other users   | 1.5               | 7      |
+| English users | 17.5              | 50     |
+
+If the dataset contains multiple rows, the visualization displays the last row by default, but multiple rows can be configured as described in the [Value options](#value-options) section. If you configure multiple rows to be displayed, the panel concatenates the text fields of each row with the numeric column titles.
```

Comment:
Can you show explicitly how you get multiple rows to be shown in the pie chart? And maybe even show an example.

Reply:
I will add images for that. I will try to keep them as generic Grafana pie as possible to avoid image maintenance.